src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableNestedClasses3.java

```diff
@@ -0,0 +1,21 @@
+/*
+UnusedLocalVariable
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+public class InputUnusedLocalVariableNestedClasses3 {
```

Comment:
Please add cases from reports:
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/bbfd2c4_2022092552/reports/diff/openjdk17/xref/home/runner/work/checkstyle/checkstyle/contribution/checkstyle-tester/repositories/openjdk17/test/langtools/tools/javac/annotations/typeAnnotations/newlocations/NestedTypes.java.html#L162,
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/bbfd2c4_2022092552/reports/diff/checkstyle/xref/home/runner/work/checkstyle/checkstyle/contribution/checkstyle-tester/repositories/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/grammar/antlr4/InputAntlr4AstRegressionUncommon4.java.html#L75, and
`@A Outer.@B Inner p2 = p1.new @B Inner();`

Reply:
Done